frontend/src/components/SchemaEdit.vue

```diff
@@ -151,24 +160,26 @@ export default {
         schemaSlug: this.schema.slug,
         body: toSend,
       });
-      if (response.status == 200) {
-        this.$router.push(`/${this.slug}`);
+      if (response.status === 200) {
+        this.$router.push({name: 'schema-view', params: {schemaSlug: this.schema.slug}});
```

Comment:
Here you are redirecting to slug of schema which was initially there, but if we change slug we'll get 404 error. And you can't just use `this.details.slug` here instead — if schema is reviewable, then this update (possibly) won't be applied immediately so there won't be a schema in database that has slug contained in `this.details.slug`. I guess to handle that it's possible to analyze response content or even have different 2xx for cases when something was updated immediately and when there was created change request.

But that also will require some changes in backend, because currently for some reason it just creates request and applies it immediately🙄

Reply:
Good point. We could just not do redirects and show nice success/error messages instead? :thinking:

Reply:
That's an option too, although I think that redirects still make sense🙃